dalek/cleaners/resource_group_servicebus_namespace_break_pairing.go

```diff
@@ -43,6 +43,10 @@ func (s serviceBusNamespaceBreakPairingCleaner) Cleanup(ctx context.Context, id
 		}
 
 		for _, config := range configs.Items {
+			if props := config.Properties; props == nil || *props.Role == disasterrecoveryconfigs.RoleDisasterRecoverySecondary {
+				log.Printf("[DEBUG] Skipping %s for %s: Role is %q", *config.Id, *namespace.Id, *props.Role)
```

Comment:
technically there's a crash here if `props` or `props.Role` is nil?

```suggestion
				role := ""
				if props != nil && props.Role != nil {
					role = *props.Role
				}
				log.Printf("[DEBUG] Skipping %s for %s: Role is %q", *config.Id, *namespace.Id, role)

```

Reply:
True, I figured it would be better to find out about that in a panic similar to line 34?

Reply:
Sorry missed this, sounds good 👍